CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0] - 2021-01-05
```

Comment:
Before we release it:

Maybe it would be good to add the separate controller setting config version annotation (https://github.com/giantswarm/giantswarm/issues/14748). Otherwise this release doesn't really result in anything useful happening in the cluster. I am happy to be unassigned and focus on admission webhook.